integration/walletextension/wallet_extension_test.go

```diff
@@ -391,15 +387,15 @@ func waitForWalletExtension(t *testing.T, walletExtensionAddr string) {
 }
 
 // Makes an Ethereum JSON RPC request and returns the response body.
-func makeEthJSONReq(t *testing.T, walletExtensionAddr string, method string, params interface{}) []byte {
+func makeEthJSONReq(walletExtensionAddr string, method string, params interface{}) []byte {
 	reqBodyBytes, err := json.Marshal(map[string]interface{}{
 		"jsonrpc": "2.0",
 		"method":  method,
 		"params":  params,
 		"id":      "1",
 	})
 	if err != nil {
-		t.Fatal(err)
+		panic(err)
```

Comment:
I'm not sure about this. The [Uber style-guide](https://github.com/uber-go/guide/blob/master/style.md#dont-panic) says: `Even in tests, prefer t.Fatal or t.FailNow over panics to ensure that the test is marked as failed.`

I do sometimes see a sim test show as green when it clearly broke (e.g. if I leave a debugger point for too long then when I eventually quit it it shows test passed).

Reply:
I've also heard arguments in the opposite direction, because `panic` gives you a stacktrace, whereas t.Fatal is opaque (no indication of stack at time of failure).